src/services/navigation/useNavLinks/useNavLinks.js

```diff
@@ -177,40 +177,48 @@ export function useNavLinks() {
     },
     treeView: {
       path: (
-        { provider = p, owner = o, repo = r, tree, ref } = {
+        { provider = p, owner = o, repo = r, tree, ref, queryParams } = {
           provider: p,
           owner: o,
           repo: r,
+          queryParams: null,
```

Comment:
You can default to an empty string `" "` instead of `null` instead of making a new var